<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSessionControlsWidget</class>
 <widget class="QWidget" name="VideoSessionControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoSessionControlsWidget</string>
  </property>
  <property name="styleSheet">
   <string>QWidget#VideoSessionControls {
	background-color: rgb(255, 255, 255);
	color: rgb(0,0,0);
}

QCheckBox {
	spacing: 3px;
}

QCheckBox::indicator {
	width: 16px;
	height: 16px;
}

QCheckBox::indicator:checked {
	image: url(:images/iconGreen.png);
}

QCheckBox::indicator:unchecked {
	image: url(:images/iconRed.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="videoCheckBox">
       <property name="text">
        <string>Video</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="audioCheckBox">
       <property name="text">
        <string>Audio</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
